docs/getting-started/getting-started-rest/01-setup-basic-syntax.mdx

```diff
@@ -31,14 +33,14 @@ Before we start writing TypeSpec code, we need to set up our development environ
 
 Once you've completed these steps, you'll have a basic TypeSpec project set up. Here's an overview of the files and directories in your TypeSpec project:
 
-```
-Project Root
-├── main.tsp
-├── tspconfig.yaml
-├── package.json
-├── node_modules/
-└── tsp-output/
-```
+{/* prettier-ignore */}
+<FileTree>
+- main.tsp
+- tspconfig.yaml
+- package.json
+- node_modules/
+- tsp-output/
```

Comment:
Should we add `openapi.yaml` under `tsp-output` as well? It's listed just a few lines below.

Reply:
yeah replicated the root one